handlers/BillTechLinkInsertHandler.php

```diff
@@ -87,7 +87,13 @@ public function messageaddCustomerDataParse(array $hook_data = array())
 		$amount = sprintf('%01.2f', -$hook_data['data']['balance']);
 		$btnPatterns = ['/%billtech_balance_btn/', '/'.$amount.'illtech_balance_btn/'];
 
-		if ($hook_data['data']['phone']) {
+		if(isset($hook_data['data']['phone'])) {
+			$phone = $hook_data['data']['phone'];
+		}else if(isset($hook_data['data']['phones'])) {
+			$phone = $hook_data['data']['phones'];
```

Comment:
Rozmawialiśmy o tym, że w przypadku ['phones'], to tutaj może być kilka numerów.

Reply:
Ale to czy jest jeden czy kilka telefonów nie robi nam różnicy, chodzi o wklejanie skróconych/nie-htmlowych linków jak tylko wysyłka idzie na jakikolwiek numer telefonu.